Lambda-School-Labs/CS10-quizzer
# Brandon/stripe

# Description

This PR adds subscriptions for users (basic $9.99, premium $29.99) and will also prevent them from signing up for multiple subscriptions at one time.

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

- Log in or create new account
- Navigate to billing section
- Choose either plan using test credentials
```
email: << valid email >>
card #: 4242 4242 4242 4242
date: 02 2020
CVC: 123
```
- After payment is successful try again
- Should return response in console letting customer know they should cancel current subscription

# Checklist:
